pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_sni.go

```diff
@@ -214,6 +218,19 @@ func (s *sni) Deploy(ctx context.Context) error {
 		accessControl.Spec = istioapisecurityv1beta1.AuthorizationPolicy{
 			Action: accessControlSpec.Action,
 			Rules:  accessControlSpec.Rules,
+			Selector: &istiov1beta1.WorkloadSelector{
+				MatchLabels: map[string]string{
```

Comment:
Are provided in the SNI Values accessible through the sni struct. No need to use constant values for the labels.